.travis.yml

```diff
@@ -4,6 +4,7 @@ dist: trusty
 
 go:
   - 1.9.x
+  - 1.10.x
```

Comment:
the change is ok. but we don't need to run it with go1.10. So just keep the change in the scope of fixing the data race.

Reply:
Got it.